core/src/main/java/nu/yona/server/messaging/service/MessageService.java

```diff
@@ -238,7 +238,13 @@ public void sendMessage(Message message, MessageDestinationDto destination)
 	{
 		MessageDestination destinationEntity = MessageDestination.getRepository().findOne(destination.getId());
 		destinationEntity.send(message);
-		MessageDestination.getRepository().save(destinationEntity);
+		MessageDestination.getRepository().saveAndFlush(destinationEntity);
+	}
+
+	@Transactional
+	public void sendMessage(Message message, MessageDestination destinationEntity)
```

Comment:
It is a bit confusing that this method does not call a flush on the `MessageDestination` repostitory as the other `sendMessage` method above. Maybe we should rename one of both?